src/main/include/commands/BalanceCommand.h

```diff
@@ -27,7 +27,8 @@ class BalanceCommand
 
   void Initialize() override {
     isFinished = false;
-    double roll = m_drive->getGyro()->GetRoll() * (M_PI / 180);
+    // double roll = m_drive->getGyro()->GetRoll() * (M_PI / 180);
```

Comment:
can change this back

Reply:
We could remove the `BalanceCommand` altogether

Reply:
Removed